Sources/SwifterSwift/SwiftStdlib/SequenceExtensions.swift

```diff
@@ -308,3 +335,14 @@ public extension Sequence where Element: AdditiveArithmetic {
         return reduce(.zero, +)
     }
 }
+
+// MARK: - Methods (Numeric)
+
+public extension Sequence where Element: Numeric {
+    /// SwifterSwift: Product of all elements in array.
```

Comment:
```suggestion
    /// SwifterSwift: Product of all elements in sequence.
```
Probably worth changing sum as well.